modules/gitops-release-notes-1-16-0.adoc

```diff
@@ -7,7 +7,7 @@
 [id="gitops-release-notes-1-16-0_{context}"]
 = Release notes for {gitops-title} 1.16.0
 
-{gitops-title} 1.16.0 is now available on {OCP} 4.12, 4.13, 4.14, 4.15, 4.16, 4.17, and 4.18.
+{gitops-title} 1.16.0 is now available on {OCP} 4.12, 4.14, 4.15, 4.16, 4.17, and 4.18.
```

Comment:
@Dhruv-Soni11 just checking that skipping 4.13 is correct?

Reply:
Yes @JoeAldinger, it was discussed in this Slack thread [here](https://redhat-internal.slack.com/archives/C087QRDA8Q3/p1748326709856579).